stac/canterbury/canterbury_2024_0.2m/rgb/2193/collection.json

```
@@ -0,0 +1,10550 @@
+    },
+    {
+      "href": "./BY19_5000_0703.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "122061bacd67327572e7dd6cc469e4d277fd805d7e0875b2ed0505d7d06687e7c0af"
+    },
+    {
+      "href": "./BY19_5000_0704.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12205ef3715de2409a6cad3c33a82d9aae29d5fef5c8d1a2779bfeded66424d9f148"
+    },
+    {
+      "href": "./BY19_5000_0705.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12204d7d2c1b34377cee6815b543a9cad2be590f5321309e854810ad6f5db48ba3c8"
+    },
+    {
+      "href": "./BY19_5000_0706.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "1220577ba7a28f96c665830492bd5fef77bb4c26c108d085852f6c9e0cb1f6adda68"
+    },
+    {
+      "href": "./BY19_5000_0707.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12205c65ae57172e1af9d697c6d5954ef813df73f26984561e2ed2b693b6bc1fc63b"
+    },
+    {
+      "href": "./BY19_5000_0708.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12205e7c11de082857dc0c2b313ea104fcc9a66ef69a9fd7a94b127757e0da88e490"
+    },
+    {
+      "href": "./BY19_5000_0709.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "1220f08b1018170ca4efbe6b7327b172545176121a9bc6a51c124221915cef716d52"
+    },
+    {
+      "href": "./BY19_5000_0710.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12202a3481fd0707412070e6b3c1ab24bf476944be6073ea3452e46c304c1254d3e5"
+    },
+    {
+      "href": "./BY19_5000_0801.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "1220179d2457a2bbc6fec4603853e52bbfcf55bcf34c191ddefa4d933598d03960db"
+    },
+    {
+      "href": "./BY19_5000_0802.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "1220328bf9ff7dcc8132d80b754958bb474f004e1efa22b89863c7aaa0dd7226b927"
+    },
+    {
+      "href": "./BY19_5000_0803.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "1220e8b99588eaebe957b68985573a41a1d3818c3a0306a6375eac6b013981e749e9"
+    },
+    {
+      "href": "./BY19_5000_0804.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12207bff1afb10b898ee39ed695ffdffe98d7e8e36315549cad8f857c1b8517f11b3"
+    },
+    {
+      "href": "./BY19_5000_0805.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12203237d71a3ed814987c3b8fa394caa7a5ff6bd868b2bad693494740c4c508058e"
+    },
+    {
+      "href": "./BY19_5000_0806.json",
+      "rel": "item",
+      "type": "application/geo+json",
+      "file:checksum": "12201bcb7ac3b3ac58a45c7b7542492
```

```suggestion
  "linz:slug": "canterbury_2024_0.2m",
```
Looks like we need to handle where the `m` is supplied and where it is not (currently only handle the latter).

---

Also looks like there is a bug in the code suggestions feature. It might commit that suggestion OK but we'd need to check.